runtime-optenv32/nlohmann-json+32/autobuild/build

```diff
@@ -0,0 +1,3 @@
+abinfo "Linking /opt/32/include/nlohmann -> /usr/include/nlohmann ..."
+install -dv "$PKGDIR"/opt/32/include
+ln -sv /usr/include/nlohmann "$PKGDIR"/opt/32/include/nlohmann
```

Comment:
Use relative paths for symlinks for avoid cross-contamination when accessing the rootfs from an external system.